src/executables/uber-time.js

```diff
@@ -8,13 +8,23 @@ import CommandExecutionService from '../services/CommandExecutionService';
 
 const service = new CommandExecutionService();
 
+
+const addressArg = program.parse(process.argv);
```

Comment:
Is there a reason why this validation can't live in `CommandExecutionService`? I thought [we decided that parameter validation should live in `CommandExecutionService`](https://github.com/jaebradley/uber-cli/pull/43#discussion_r122726742)?